packages/terra-modal/src/Modal.jsx

```diff
@@ -4,6 +4,8 @@ import { Portal } from 'react-portal';
 import 'terra-base/lib/baseStyles';
 import ModalContent from './ModalContent';
 
+const zIndexes = ['6000', '8000'];
```

Comment:
Just curious, how did we decide on these two z-index values?

Reply:
6000 was the previous value, the popup is at 7000, so i went an additional layer to 8000.

Reply:
I vote 9001, for no reason...
![](https://media.giphy.com/media/tPKoWQJk3cEbC/giphy.gif)